### DISCUSSION TITLE
Closed a PR. But a new PR opened (from the same branch) on subsequent run

### DISCUSSION THREAD

**Discussion by klippx** [renovatebot/renovate#21627] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.0.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

Not sure

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

When I run renovate and there is a closed PR for a renovate branch, renovate should not open the PR again, since this PR has already been seen, reviewed and closed.

### What you tried so far.

I have this action:
```yml
    - name: Self-hosted Renovate
      uses: renovatebot/github-action@v36.0.0
      with:
        configurationFile: .github/renovate.json
        token: ${{ secrets.GH_PAT }}
```

And this config:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "username": "renovate-release",
  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
  "onboarding": false,
  "platform": "github",
  "labels": ["dependencies"],
  "extends": ["config:js-app", ":disableDependencyDashboard"],
  "prHourlyLimit": 0,
  "prConcurrentLimit": 4
}
```

I run the job. It opened this PR for me `Update Node.js to v18.16.0 #1946`. I take a look at it, decide that I don't want it since I am fine on 18.15 (due to other dependency not being updated to work for 18.16 just yet). 

So I close this PR, assuming that this would tell renovate I am not interested in this particular update.

I run the job again. Now it opens a new PR for the same branch again: `Update Node.js to v18.16.0 #1948`. 

Why is this happening?

Is there a way to make it stop?

Also, it opened another for `v19` which is not LTS. I want to permanently ignore node `v19`, is there a way to do that? I was hoping it would be as simple as closing the PR 😃 

### Relevant debug logs

<details><summary>Info Logs</summary>

Without debug, can update if needed:

```
Status: Downloaded newer image for renovate/renovate:slim
 INFO: Repository started (repository=myrepo)
       "renovateVersion": "35.54.0"
 INFO: Dependency extraction complete (repository=myrepo, baseBranch=main)
       "stats": {
         "managers": {
           "asdf": {"fileCount": 1, "depCount": 2},
           "dockerfile": {"fileCount": 2, "depCount": 2},
           "github-actions": {"fileCount": 17, "depCount": 51},
           "npm": {"fileCount": 17, "depCount": 441}
         },
         "total": {"fileCount": 37, "depCount": 496}
       }
 INFO: GitHub GraphQL datasource: skipping empty item (repository=myrepo)
       "packageName": "actions/labeler",
       "baseUrl": "https://api.github.com/"
 INFO: Workflows update rejection - aborting branch. (repository=myrepo, branch=renovate/renovatebot-github-action-36.x)
 INFO: PR created (repository=myrepo, branch=renovate/node-18.x)
       "pr": 1948,
       "prTitle": "Update Node.js to v18.16.0"
 INFO: Branch created (repository=myrepo, branch=renovate/major-apollo-graphql-packages)
       "commitSha": "08a43a59ef305b7682a4de796b17ce1ccf10e6f9"
 INFO: PR created (repository=myrepo, branch=renovate/major-apollo-graphql-packages)
       "pr": 1949,
       "prTitle": "Update Apollo GraphQL packages to v3 (major)"
 INFO: Branch created (repository=myrepo, branch=renovate/node-19.x)
       "commitSha": "30e63cbb78f7b9fd190eabfd09bbaf8293f37ae4"
 INFO: PR created (repository=myrepo, branch=renovate/node-19.x)
       "pr": 1950,
       "prTitle": "Update Node.js to v19"
 INFO: Repository finished (repository=myrepo)
```

</details>

<details><summary>Debug Logs</summary>

https://gist.github.com/klippx/b9b3975b4849da71c8b516048bf06572

</details>

**Comment by rarkins** [comment]
Please read https://docs.renovatebot.com/key-concepts/pull-requests/#immortal-prs and reply if any further questions

**Reply by klippx** [reply]
So the PR does not have this entry:
> 👻 Immortal: This PR will be recreated if closed unmerged. Get [config help](https://github.com/renovatebot/renovate/discussions) if that's undesired.

So it is not an "immortal PR". It has only this:

> Configuration
> 📅 Schedule: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).
> 🚦 Automerge: Disabled by config. Please merge this manually once you are satisfied.
> ♻ Rebasing: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.
> 🔕 Ignore: Close this PR and you won't be reminded about this update again.

So according to this read https://docs.renovatebot.com/key-concepts/pull-requests/#normal-prs I guess that if I closed it then it shouldnt reopen, just like I expected, since I closed a "unique PR" in this case the unique title was `Update Node.js to v18.16.0`?

**Reply by rarkins** [reply]
Please copy/paste in the debug logs for this branch for the run when the PR was reopened.

**Reply by klippx** [reply]
The debug logs are 10x bigger than the allowed limit, but here is a gist with the contents: https://gist.github.com/klippx/b9b3975b4849da71c8b516048bf06572

In this logs we see it opened a new PR `Update Node.js to v18.16.0`, although we already have three closed ones with exactly the same title. 
And same with `Update Node.js to v19`.

**Reply by rarkins** [reply, marked as answer]
You have configured the `username` field erroneously in your file config. Remove that setting

**Reply by klippx** [reply]
Well, that fixed it! Thank you very much!
